openpathsampling/engines/openmm/topology.py

```diff
@@ -1,95 +1,14 @@
-import mdtraj as md
-import numpy as np
-import pandas as pd
```

Comment:
@dwhswenson I assumed I could get away with not supporting `openpathsampling.engines.openmm.topology.md` etc anymore. (technically an API break, and can be added to the `__getattr__` solution if you still think we should keep this API in place)

Reply:
Not an API break: our API promise is for the first level (breadth-first search) where it can be imported. Since MDTraj can be imported from outside of OPS, it's not in our scope.

The entire importable API for 1.0 is listed [in this file](https://github.com/openpathsampling/openpathsampling/blob/master/openpathsampling/tests/test_data/apis/ops1x0_api.txt), and checking that we haven't violated it is literally the first test in each CI run (mainly because `test_api` comes first alphabetically). This is the official list of the "documented API" -- and because it is in unit tests, we're guaranteed to not to break it.

If you're curious, the script I used to generate that file is in my [define_api](https://github.com/dwhswenson/define_api) repo. It may have also involved some manual editing of the output file, but most of it should have been generated by `./define_api.py --runtype=in_api --api-file=ops_api_dirs.txt openpathsampling`.

However, it looks like we didn't guarantee `MDTrajTopology` in that. This is probably because it was not imported into a package, as it should have been (and I overlooked it when doing the `--runtype=not_in_api` run). Moving it now will put it on the API list when I regenerate that for 1.5 (which will be after I generate the versions for the tags at 1.1, 1.2, 1.3, and 1.4).